pantalaimon/daemon.py

```diff
@@ -1239,6 +1239,10 @@ async def profile(self, request):
         try:
             content = await self._map_decrypted_uri("avatar_url", content, request, client)
             return await self.forward_to_web(request, data=json.dumps(content), token=client.access_token)
+        except ClientConnectionError as e:
+            return web.Response(status=500, text=str(e))
+        except KeyError:
+            return await self.forward_to_web(request, token=client.access_token)
```

Comment:
You can combine those last two as
```python
except (KeyError, ValueError):
    return await self.forward_to_web(request, token=client.access_token)
```

Of course change this to the correct error class instead of `ValueError` once that's changed.